util/drakeGeometryUtil.cpp

```diff
@@ -564,14 +609,27 @@ Eigen::Matrix<typename Derived::Scalar, 3, 3> vectorToSkewSymmetric(const Eigen:
   return ret;
 }
 
+template <typename DerivedA, typename DerivedB>
+Eigen::Matrix<typename DerivedA::Scalar, 3, Eigen::Dynamic> dcrossProduct(
```

Comment:
Not quite the dcross from Matlab, but more convenient I think because of the da and db inputs. Also, couldn't call it dcross because otherwise there would have been a name clash with the dcross member variable in RigidBodyManipulator...